Formula/zsh-completions.rb

```diff
@@ -14,6 +14,9 @@ class ZshCompletions < Formula
 
   def install
     inreplace "src/_ghc", "/usr/local", HOMEBREW_PREFIX
+    # We install this into `pkgshare` to avoid conflicts
+    # with completions installed by other formulae. See:
+    #   https://github.com/Homebrew/homebrew-core/pull/126586
```

Comment:
```suggestion
```
Listing this PR will result in audit failures once it's closed.

Reply:
No, it won't. We allow [references to Homebrew PRs](https://github.com/Homebrew/brew/blob/140d4444621c7df81e8a146429eeec1bfe3c5b9d/Library/Homebrew/formula_auditor.rb#L838-L839).
```ruby
        # Do not trigger for self references
        next if "#{owner}/#{repo}" == formula.tap.remote_repo || owner == "Homebrew"
```

Reply:
Had one of those "not enough coffee yet" morning moments and remembered right after I commented 😅